sympy/printing/latex.py

```diff
@@ -2452,6 +2477,12 @@ def latex(expr, fold_frac_powers=False, fold_func_brackets=False,
     gothic_re_im : boolean, optional
         If set to ``True``, `\Re` and `\Im` is used for ``re`` and ``im``, respectively.
         The default is ``False`` leading to `\operatorname{re}` and `\operatorname{im}`.
+    decimal_separator : string, optional
+        Specifies what separator to use to separate the whole and fractional parts of a
+        floating point number as in `2.5` (for the default, "period") or `2{,}5`
```

Comment:
If I understand correctly using a single backtick leads to LaTeX rendering, which is OK, but it is not so clear for the user that the `{}` are added as well. Not sure what is the best option.

I think that the () can be removes though.

For consistency I think all options (period, comma) should be in double backticks, which will make it print as source code.

Reply:
I'm not sure about the {} as well. It was something to do with the spacing being strange around the commas during the latex rendering. 

 `2.5` for the default, ``period`` or `2{,}5` when ``comma`` is specified. Lists, sets, and tuple are printed with semicolon separating the elements when ``comma`` is chosen. For example, [1; 2; 3] when comma is chosen and [1,2,3] for when ``period`` is chosen).